db/migrate/20231108095514_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
```

Comment:
- [ ] Great work so far! Just a quick note: when creating a foreign key with a column name that differs from the table name, like in line `4`, Rails may not know which table to refer to by default. To resolve this, it's important to explicitly specify the table name, as demonstrated in the code snippet below:
```rb
 t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Thank you, This has been updated accordingly